<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseParams</class>
 <widget class="QWidget" name="ChooseParams">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChooseParams</string>
  </property>
  <widget class="QGroupBox" name="criteriaGB">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>161</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Stop criteria by:</string>
   </property>
   <widget class="QRadioButton" name="xcritRB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="fcritRB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>f</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="gradcritRB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>gradient</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="methodGB">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Method:</string>
   </property>
   <widget class="QRadioButton" name="frmethodRB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>feltcher reeves</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="rsmethodRB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>random search</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_2</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>351</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Area</string>
   </property>
   <widget class="QDoubleSpinBox" name="right2DSB">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="left1DSB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="left2DSB">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>right1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>left1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>left2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>right2</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="right1DSB">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="plotButton">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>520</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Plot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Params for 2d optimization</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Functions:</string>
   </property>
   <widget class="QRadioButton" name="sqsmRB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>x^2 + y^2</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="rosenbrokRB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Rosenbrok</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="sincosRB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>112</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>sin(x) cos(y)</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="expRB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>151</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>exp(-x - y) + x + y</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup_3</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup_3"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
